### PR TITLE
[Ready]Better crate options

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Makes most crates now a option with plasteel
Radiation crates now contain radiation!
New Radiation Body bag is now a thing that only holds 1 person

Freezers cost 8 plasteel do to helping store orgains
Radiation Crates cost 20 plasteel for holding back radiation

## Why It's Good For The Game

Theirs times when medical is still swamped with dead and just run out of body bags or need to print them off do to the box being lost.
As for crate crafting, only having the gray ones seems limited and not helpfull for making storge
Radiation containment should not be just to space dead bodys, or throw your tool box into a deep fryer to contain the radiation. Thus the new radition containment flags on things that are ment to store such things are added

## Changelog
:cl:
add: Most crate types can now be made, some costing more do to function over fashion 
/:cl:

### CODE REVIEW

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -75,12 +75,20 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 	export_price = 5000
 
+/datum/techweb_node/aftercare
+	id = "aftercare"
+	display_name = "Aftercare"
+	description = "What you use when its to late."
```
Comment (dapnee) [inline code/modules/research/techweb/all_nodes.dm]:
```suggestion
	description = "What you use when it's too late."
```

Comment (dapnee) [reply code/modules/research/techweb/all_nodes.dm]: Only grammar issue I see, good job.

**Review by dapnee — COMMENTED** [review]
A little grammar issue here.

File: code/modules/research/techweb/all_nodes.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/research/techweb/all_nodes.dm]: Huh... forgot the to vs too

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 2]
```diff
@@ -154,8 +154,21 @@ GLOBAL_LIST_INIT(metal_recipes, list ( \
...
+		new /datum/stack_recipe("freezer crate", /obj/structure/closet/crate/freezer, 8, time = 50, one_per_turf = 1, on_floor = 1), \
+		new /datum/stack_recipe("blood bag crate", /obj/structure/closet/crate/freezer/blood, 8, time = 50, one_per_turf = 1, on_floor = 1), \
+		new /datum/stack_recipe("surplus limbs crate", /obj/structure/closet/crate/freezer/surplus_limbs, 8, time = 50, one_per_turf = 1, on_floor = 1), \
+		new /datum/stack_recipe("radiation containment crate", /obj/structure/closet/crate/radiation, 20, time = 50, one_per_turf = 1, on_floor = 1), \
+		new /datum/stack_recipe("hydroponics crate", /obj/structure/closet/crate/hydroponics, 5, time = 50, one_per_turf = 1, on_floor = 1), \
+		new /datum/stack_recipe("engineering crate", /obj/structure/closet/crate/engineering, 5, time = 50, one_per_turf = 1, on_floor = 1), \
+		new /datum/stack_recipe("eletrical crate", /obj/structure/closet/crate/engineering/electrical, 5, time = 50, one_per_turf = 1, on_floor = 1), \
+		new /datum/stack_recipe("RCD storge crate", /obj/structure/closet/crate/rcd, 5, time = 50, one_per_turf = 1, on_floor = 1), \
```
Comment (Ghommie) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]:
```suggestion
		new /datum/stack_recipe("RCD storage crate", /obj/structure/closet/crate/rcd, 5, time = 50, one_per_turf = 1, on_floor = 1), \
```

File: code/game/objects/structures/crates_lockers/closets/bodybag.dm [hunk 3]
```diff
@@ -100,3 +100,12 @@
 			if(isliving(A))
 				to_chat(A, "<span class='userdanger'>You're suddenly forced into a tiny, compressed space!</span>")
 		qdel(src)
+
+/obj/structure/closet/body_bag/containment
+	name = "containment body bag"
+	desc = "A folded heavy body bag designed for the storage and transportation of cadavers with heavy radiation."
```
Comment (Ghommie) [inline code/game/objects/structures/crates_lockers/closets/bodybag.dm:106]:
```suggestion
	desc = "A folded heavy body bag designed for the storage and transportation of heavily irradiated cadavers."
```

File: code/game/objects/items/bodybag.dm [hunk 4]
```diff
@@ -81,3 +81,14 @@
...
+// Containment bodybag
+
+/obj/item/bodybag/containment
+	name = "radiation containment body bag"
+	desc = "A folded heavy body bag designed for the storage and transportation of cadavers with heavy radiation."
+	icon = 'icons/obj/bodybag.dmi'
+	icon_state = "radbodybag_folded"
+	unfoldedbag_path = /obj/structure/closet/body_bag/containment
+	w_class = WEIGHT_CLASS_NORMAL
+	item_flags = NO_MAT_REDEMPTION | RAD_NO_CONTAMINATE
```
Comment (Ghommie) [inline code/game/objects/items/bodybag.dm]: radiation flags go to `rad_flags`.

**Review by Ghommie — CHANGES_REQUESTED** [review]
This won't stop corpses and other contained hazards from spawning radiation waves and irradiating the outside atoms anyway.
I'll work on to implement the code to stop rad waves for protected contents, but until then this is going to pretty much not work.

File: code/game/objects/items/bodybag.dm [hunk 5]
```diff
@@ -81,3 +81,14 @@
 		return
 	loc.visible_message("<span class='warning'>[user] suddenly appears in front of [loc]!</span>", "<span class='userdanger'>[user] breaks free of [src]!</span>")
 	qdel(src)
+
+// Containment bodybag
+
+/obj/item/bodybag/containment
+	name = "radiation containment body bag"
+	desc = "A folded heavy body bag designed for the storage and transportation of cadavers with heavy radiation."
+	icon = 'icons/obj/bodybag.dmi'
```
Comment (Ghommie) [inline code/game/objects/items/bodybag.dm]: Remove, redundant.

File: code/game/objects/items/bodybag.dm [hunk 6]
```diff
@@ -86,9 +85,8 @@
...
 /obj/item/bodybag/containment
 	name = "radiation containment body bag"
-	desc = "A folded heavy body bag designed for the storage and transportation of cadavers with heavy radiation."
-	icon = 'icons/obj/bodybag.dmi'
+	desc = "A folded heavy body bag designed for the storage and transportation of heavily irradiated cadavers."
 	icon_state = "radbodybag_folded"
 	unfoldedbag_path = /obj/structure/closet/body_bag/containment
 	w_class = WEIGHT_CLASS_NORMAL
-	item_flags = NO_MAT_REDEMPTION | RAD_NO_CONTAMINATE
+	rad_flags = NO_MAT_REDEMPTION | RAD_NO_CONTAMINATE
```
Comment (Ghommie) [inline code/game/objects/items/bodybag.dm]:
```
	item_flags = NO_MAT_REDEMPTION
	rad_flags = RAD_PROTECT_CONTENTS
```

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/game/objects/items/bodybag.dm (hunk 6 above)
Comment (Trilbyspaceclone) [reply code/game/objects/items/bodybag.dm]: item flag isnt needed as its in the sub-type

**Review by Linzolle — APPROVED** [review]

**Review by Ghommie — APPROVED** [review]

**Pull request merged**